base-common/duneci-standard-test

```diff
@@ -35,5 +35,9 @@ export OMPI_MCA_mpi_yield_when_idle=1
 set -x
 ${DUNECONTROL} --current "${@}" vcsetup
 ${DUNECONTROL} --current "${@}" configure
-${DUNECONTROL} --current "${@}" make ${parallel_opts} all build_tests
+${DUNECONTROL} --current "${@}" make ${parallel_opts} all
+if [[ -v DUNECI_TEST_INSTALL ]] ; then
+    ${DUNECONTROL} --current "${@}" make ${parallel_opts} install
+fi
+${DUNECONTROL} --current "${@}" make ${parallel_opts} build_tests
```

Comment:
I changed the order because `make install` will typically be a lot faster than building the tests. Moreover, for most modules CMake will finish building the library before starting to build the tests.

The variable check should definitely test for `0`, I'll change that.